packages/adapter-vercel/index.js

```diff
@@ -544,6 +549,24 @@ async function create_function_bundle(builder, entry, dir, config) {
 		)
 	);
 
+	const server_assets = builder.getServerAssets();
```

Comment:
This needs to be called conditionally - the peer dependency of the adapter includes a version that does not contain this function yet. The alternative would be to have another major version bump (don't know if we need that if we can just call this conditionally)